pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -98,6 +98,25 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		disableFeatureGates()
 	})
 
+	It("when spec validator pass, should allow", func() {
+		ar, err := newAdmissionReviewForVMICreation(newBaseVmi())
+		Expect(err).ToNot(HaveOccurred())
+
+		admitter := &VMICreateAdmitter{ClusterConfig: config, SpecValidators: []ValidateSpec{validateStub()}}
+		resp := admitter.Admit(context.Background(), ar)
+		Expect(resp.Allowed).To(BeTrue())
+	})
+	It("when spec validator fail, should reject", func() {
+		expectedStatusCause := []metav1.StatusCause{{Type: "test", Message: "test", Field: "test"}}
```

Comment:
nit: Since its a slice, its common to have a plural name for it:
```suggestion
		expectedCauses := []metav1.StatusCause{{Type: "test", Message: "test", Field: "test"}}
```

Reply:
Fixed